supervisor/addons/addon.py

```diff
@@ -679,23 +679,23 @@ async def write_stdin(self, data) -> None:
         except DockerError as err:
             raise AddonsError() from err
 
-    async def _snapshot_command(self, command: str) -> None:
+    async def _backup_command(self, command: str) -> None:
         try:
             command_return = await self.instance.run_inside(command)
             if command_return.exit_code != 0:
                 _LOGGER.error(
-                    "Pre-/Post-Snapshot command returned error code: %s",
+                    "Pre-/Post backup command returned error code: %s",
                     command_return.exit_code,
                 )
                 raise AddonsError()
         except DockerError as err:
             _LOGGER.error(
-                "Failed running pre-/post-snapshot command %s: %s", command, err
+                "Failed running pre-/post backup command %s: %s", command, err
             )
             raise AddonsError() from err
 
-    async def snapshot(self, tar_file: tarfile.TarFile) -> None:
-        """Snapshot state of an add-on."""
+    async def backup(self, tar_file: tarfile.TarFile) -> None:
```

Comment:
I think that could stay as `snapshot` to match with `restore` in a backup process?